test/test_go.rb

```diff
@@ -1016,15 +1003,13 @@ def test_execute_plus_flag
     tmux.send_keys :Tab
     execute.call
 
-    tmux.send_keys :Enter
-    tmux.prepare
-    readonce
-
-    assert_equal [
-      %(foo bar/foo bar/bar/bar),
-      %(123 456/foo bar/456/bar),
-      %(123 456 foo bar/foo bar/456 bar/bar)
-    ], File.readlines(output).map(&:chomp)
+    wait do
+      File.exist?(output) && File.readlines(output).map(&:chomp) == [
+        %(foo bar/foo bar/bar/bar),
+        %(123 456/foo bar/456/bar),
+        %(123 456 foo bar/foo bar/456 bar/bar)
+      ]
```

Comment:
Oh, this reads much better.

Reply:
I think you mean the line breaks inside the brackets: I'll go ahead and add them to 940 and 966 :+1: